### ISSUE TITLE
HW Folders

### ISSUE THREAD

**Issue by BedirT** [NAU-ACM/ACM-ICPC-Lab-HW#1]
Guys you need to open a folder for you to put your Algorithm Homeworks. (See the example for how to do readme.md file)

**Comment by alashow** [comment]
folder names should be our names/usernames or what?

**Comment by BedirT** [comment]
Folder names will be your usernames - and inside the folder add your names in readme

**Comment by Nadide** [comment]
Just to make sure, we will open a folder under this repo with our usernames and add only readme file which includes all weeks' assignments, right?

**Comment by BedirT** [comment]
Yes, but they should add their assignments too . OK I will create one for me as an example.

**Comment by BedirT** [comment]
OK, I put my folder, please see it on the main repo page

**Comment by atukenov** [comment]
Guys please make design same as MMM Folder (Michael's), it will be easier to check, I will make blank checked. So you will see did I check you hw or not, Thanks

**Comment by BedirT** [comment]
@atukenov So we should leave the checkboxes empty ?

**Comment by BedirT** [comment]
Also for everyone, do not skip some of the homework in the ReadMe please. @MichaelMMeskhi you have missing questions. Others have too, I just updated so you can copy mine. @atukenov you should check the question amount too when you checking hw.

**Comment by atukenov** [comment]
@BedirT Yes

**Comment by BedirT** [comment]
@atukenov Should we mark our homework as done or will you do it ?

**Comment by atukenov** [comment]
@BedirT I will do it.

**Comment by atukenov** [comment]
Guys check Nadide Abla's Folder, can you do something like that. It will be easy for me and for you too. Like add link in read the source of your code)
